spec/unit/peep_spec.rb

```diff
@@ -0,0 +1,27 @@
+require 'peep'
+
+describe Peep do
+
+  describe '.create' do
+
+    it 'allows you to create a message' do
+      expect{ Peep.create(message: 'How is everyone doing?', posted: DateTime.now.strftime('%d/%m/%_Y %_H:%_M')) }.to change{ Peep.all.length }.by (1)
```

Comment:
could you pass a hardcoded date here? (would shorten the line as well) - or you could set a `date = DateTime.now.strftime('%d/%m/%_Y %_H:%_M')` variable at the top of the tests and use that for each of the tests in this file